src/methods/using_methods/hard/q5/Using5.java

```diff
@@ -0,0 +1,39 @@
+package methods.using_methods.hard.q5;
+
+/*
+Write a Java Program called Using5 that asks users what year they are in university.
+Write a method called designationCalc that accepts that integer as an input parameter and returns a string designating them.
+If the user is in their first year, they're freshmen, if the user is in their second year, they're a sophomore,
+if the user is in their third year, they're a junior, and if the user is in their fourth year or above, they're a senior.
+If the user is not one of these options, the year is unknown.
+
+Sample output:
+Please enter what year you are in university:
+2
+Your designation is a sophomore
+ */
+
+import java.util.Scanner;
+public class Using5
+{
+    public static void main( String[] args )
+    {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Please enter what year you are in university:");
+        int currYear = input.nextInt();
+        System.out.println("Your designation is a " + designationCalc(currYear));
+    }
+    public static String designationCalc(int a)
+    {
```

Comment:
I think there should also be an invalid input check if the year is 0 or negative